data-sources/us-tiger/convert.sh

```diff
@@ -0,0 +1,50 @@
+#!/bin/bash
+
+set +x
+
+INPATH="/tmp/tiger2018raw"
+INREGEX='_([0-9]{5})_edges.zip'
+NUM_EXPECTED=3233
+
+OUTPATH="/tmp/tiger2018parsed"
+mkdir -p "$OUTPATH"
+
+WORKPATH="/tmp/tiger2018work"
+mkdir -p "$WORKPATH"
```

Comment:
I'd like to avoid hardcoded paths. How about changing the script to take two mandatory parameters 'inpath' and 'outpath' and then create a temporary work path under outpath.